Form-Controls/index.html

```diff
@@ -13,15 +13,57 @@
     <h1>Product Pick</h1>
   </header>
   <main>
-  <form>
+  <form class="form" action="reesults.html" method="GET">
```

Comment:
Nice Code!
I like to share some suggestion with you. Based on our criteria, we don't need action in in this Form. However if you test your Form you would see <page not found> after pressing submit.